.github/workflows/main.yml

```diff
@@ -14,13 +14,19 @@ jobs:
         uses: actions/checkout@v2
         with:
           lfs: false
+      - name: build latest runc # https://github.com/actions/virtual-environments/issues/2698
```

Comment:
You are referring to a different issue here than in the commit message. Was that intentional?

Reply:
The issues in the commit message point to this issue's comment as a workaround.

Reply:
Ah yes, I should have looked more closely. :)

Reply:
I updated the yaml comment as well as the commit message to say this explicitly. Hope its better now :)